internal/core/common/router/router.go

```diff
@@ -0,0 +1,20 @@
+package router
+
+import (
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+)
+
+func SetupRouter() *chi.Mux {
+	r := chi.NewRouter()
+
+	r.Get("/health", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Not a good idea.
For health we will add general Health API into gRPC. Let's do it in next PRs. Remove this router